test/unit/controllers/marriage-details.js

```diff
@@ -77,6 +94,19 @@ describe('controllers/marriage-details', function() {
             .and.to.have.deep.property('args[0][1]').that.is.an('object').and.has.property('showAll').that.is.false
         )
       );
+
+      describe('with "full-details" role', () => {
+        beforeEach(() => {
+          req.headers['X-Auth-Roles'] = role;
```

Comment:
The header name should be lowercase: https://github.com/nodejs/node-v0.x-archive/issues/1954

Reply:
Fixed.